mithril-aggregator/src/services/epoch_service.rs

```diff
@@ -112,6 +130,8 @@ struct EpochData {
     current_signers: Vec<Signer>,
     next_signers: Vec<Signer>,
     signed_entity_config: SignedEntityConfig,
+    total_spo: u32,
+    total_stake: u64,
```

Comment:
You could create type aliases here for better readability

Reply:
Done. ✅